packages/server/jest.config.js

```diff
@@ -9,8 +9,14 @@ module.exports = {
     }
   },
   transform: {
-    ...tsjPreset.transform
+    '\\.(gql|graphql)$': 'jest-transform-graphql',
+    ...tsJestPresets.jsWithBabel.transform
```

Comment:
In our current `tsconfig.js`, `allowJs` is not set so we're using the default `false`. Instead, `js` is transpiled with `babel` by `webpack`. Therefore, I believe this config more accurate represents our build config.